CHANGELOG.md

```diff
@@ -9,6 +9,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - `requirements-splitter.py`
 ### Changed
 - Test system modified
+- JSON format value changed from `string` to `number`
+- YAML format value changed from `string` to `number`
+- Pickle format value changed from `string` to `number`
```

Comment:
Isn't it better if we use `int` instead of `number`?

Reply:
the weight value is not always an integer.